lib/honeybadger/metric.rb

```diff
@@ -0,0 +1,47 @@
+module Honeybadger
+  class Metric
+    attr_reader :name, :attributes, :sampled
+
+    def self.metric_type
+      name.split('::').last.downcase
+    end
+
+    def self.signature(metric_type, name, attributes)
```

Comment:
Since this includes all attribute names and values, could get this string get large? Would it make sense to return a hash of the string rather than the full string?

Reply:
Ah yes, good idea!